platform/openide.nodes/src/org/openide/nodes/PropertySupport.java

```diff
@@ -322,14 +485,13 @@ public void setValue(T val)
         * @return the property editor or <CODE>null</CODE> if there should not be
         *    any editor.
         */
+        @Override
         public PropertyEditor getPropertyEditor() {
             if (propertyEditorClass != null) {
                 try {
-                    return propertyEditorClass.newInstance();
-                } catch (InstantiationException ex) {
+                    return propertyEditorClass.getDeclaredConstructor().newInstance();
+                } catch (InstantiationException | IllegalAccessException | NoSuchMethodException | SecurityException | IllegalArgumentException | InvocationTargetException ex) {
```

Comment:
some of them can be reduced to a `ReflectiveOperationException` ?

Reply:
Could be. Just accepted whatever the IDE offered to convert to Multicatch.